tests/decode/test5_corner_cases/src/main.c

```diff
@@ -1376,6 +1378,91 @@ void test_prelude(void)
 	zassert_equal(ZCBOR_SUCCESS, cbor_decode_Prelude(prelude_payload1, sizeof(prelude_payload1), &result, &num_decode), NULL);
 }
 
+void test_cbor_bstr(void)
```

Comment:
Please add a comment explaining what this test verifies.

Reply:
Added